src/rp2/plugin/accounting_method/lifo.py

```diff
@@ -59,8 +60,13 @@ def seek_non_exhausted_acquired_lot(
 
         if selected_acquired_lot_amount > ZERO and selected_acquired_lot:
             lot_candidates.clear_partial_amount(selected_acquired_lot)
+            if selected_acquired_lot_amount > taxable_event_amount:
```

Comment:
Could you clarify this bit?

Reply:
If the selected lot will not be entirely consumed by the current transaction, add it back to the top of the heap.

Reply:
Same as HIFO.